addons/mail/static/src/core/web/activity_list_popover_item.js

```diff
@@ -76,11 +76,22 @@ export class ActivityListPopoverItem extends Component {
         return this.props.activity.state !== "done" && !this.hasFileUploader;
     }
 
+    get hasCancelButton() {
+        return this.props.activity.state !== "done";
```

Comment:
Ah, that's a good point :sweat_smile: 
In that case, the code for `hasCancelButton` will now become identical to that of `hasEditButton`.

To reduce redundancy, should I just change the condition to display the Cancel button to `t-if="hasEditButton"`.
I'm asking because I realize the name can be a bit misleading...(displaying cancel button if 'hasEditButton').
Or do I keep a separate method `hasCancelButton` as it is even if it is identical?

Reply:
I think keeping a separate method is better :slightly_smiling_face: 
or we could make a single method and call it something like `showCancelEditButtons` but that seems a bit too much and maybe not that convenient if we want to edit only one of the 2 actions :thinking: 

I'd say, it's better to keep them separated and we'll see what the reviewer think about it :smile: 

Thanks for updating!
Cheers!

Reply:
Yes I thought so! Kept them separated :+1: